tests/test_structure/test_structure.py

```diff
@@ -0,0 +1,43 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.structure.structures_fixtures import (
+    structure_query,
+    expected_structure_query_response,
+    structure_query_non_existant_web_id,
+    expected_error_non_existant_web_id,
+    structure_query_invalid_web_id,
+    expected_error_invalid_web_id
+)
+
+
+class TestStructure(BaseTestCase):
+
+    def test_structure(self):
+        """
+        Test that an admin can get a single office structure by
+        supplying its webId
+        """
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            structure_query,
+            expected_structure_query_response
+        )
+
+    def test_structure_non_existant_web_id(self):
+        """
+        Test for office structure with non existant webId
```

Comment:
What are we testing here? Please be specific in your description. Same to in line 37 below